.rubocop.yml

```diff
@@ -1,515 +1,9 @@
-Style/AccessModifierIndentation:
```

Comment:
Because this file is clearly outdated as seen from the output when running `cookstyle` with this file in place:

```
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/AccessModifierIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/AlignArray has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/AlignHash has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/AlignParameters has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/BlockEndNewline has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/CaseIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/ClosingParenthesisIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/CommentIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/DotPosition has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/ElseAlignment has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EmptyLineBetweenDefs has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EmptyLines has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EmptyLinesAroundAccessModifier has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EmptyLinesAroundBlockBody has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EmptyLinesAroundClassBody has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EmptyLinesAroundModuleBody has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EmptyLinesAroundMethodBody has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/EndOfLine has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/ExtraSpacing has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/InitialIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/FirstParameterIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/IndentationConsistency has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/IndentationWidth has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/IndentArray has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/IndentAssignment has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/IndentHash has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/LeadingCommentSpace has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/MultilineBlockLayout has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/MultilineMethodCallIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/MultilineOperationIndentation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/RescueEnsureAlignment has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceBeforeFirstArg has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAfterColon has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAfterComma has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAfterMethodName has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAfterNot has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAfterSemicolon has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceBeforeBlockBraces has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceBeforeComma has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceBeforeComment has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceBeforeSemicolon has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceInsideBlockBraces has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAroundBlockParameters has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAroundEqualsInParameterDefault has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAroundKeyword has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceAroundOperators has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceInsideBrackets has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceInsideHashLiteralBraces has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceInsideParens has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceInsideRangeLiteral has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/SpaceInsideStringInterpolation has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/Tab has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/TrailingBlankLines has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Style/TrailingWhitespace has the wrong namespace - should be Layout
/Users/sme/src/sensu-go-chef/.rubocop.yml: Lint/Eval has the wrong namespace - should be Security
```
I'm assuming that we don't need this anymore if we use `cookstyle` instead of rubocop directly, but I would assume that the settings in this file need to agree with `cookstyle` applied changes.

I also usually favor the method of using the `delivery local` testing commands for consistency with the pattern set in all Chef supported community cookbooks and this will allow eventually switching to that.

Perhaps @jjasghar or @tas50 has more info on what the latest recommendation is for partner cookbooks?

Reply:
makes sense.